compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -2291,16 +2291,9 @@ impl<'a> Parser<'a> {
                         .span_label(else_span, "expected an `if` or a block after this `else`")
                         .span_suggestion(
                             cond.span.shrink_to_lo(),
-                            "add an `if` if this is the condition to an chained `if` statement after the `else`",
+                            "add an `if` if this is the condition to an chained `else if` statement",
```

Comment:
Grammar suggestion:
```suggestion
                            "add an `if` if this is the condition to a chained `else if` statement",
```
or
```suggestion
                            "add an `if` if this is the condition for a chained `else if` statement",
```
or maybe
```suggestion
                            "add an `if` if this is the condition of a chained `else if` statement",
```

Reply:
Whoops, thanks for catching this grammar nit. Also, I like "of a", thanks for the suggestion!